addons/spike/XEH_postInit.sqf

```diff
@@ -20,4 +20,25 @@ if (hasInterface) then {
 
     GVAR(activeCamera) = objNull;
     GVAR(projectileHashMap) = createHashMap; // used in the seeker to get the namespace for the associated projectile
+
+    GVAR(launcherWeapons) = createHashMapFromArray [[QGVAR(launcher), true], [QGVAR(launcher_olive), true]]; // Used to quickly determine whether the current weapon is a Spike launcher
+
+    // Define the B&W ppEffect for the visual seeker view mode
+    GVAR(ppEffectBW) = ppEffectCreate ["ColorCorrections", 1500];
+    GVAR(ppEffectBW) ppEffectAdjust [1, 1, 0, [0, 0, 0, 0], [1, 1, 1, 0], [0.33, 0.33, 0.33, 0]];
+    GVAR(ppEffectBW) ppEffectCommit 0;
+    GVAR(ppEffectBW) ppEffectEnable false;
+
+    // Stores launcher view mode, to auto-set the missile's post-launch
+    GVAR(visionMode) = "normal";
+    player addEventHandler ["VisionModeChanged", {
```

Comment:
```suggestion
    ace_player addEventHandler ["VisionModeChanged", {
```
actually I'm not sure; will have problems with respawn/zeus-rc?

Reply:
maybe `["visionMode", {...}, false] call CBA_fnc_addPlayerEventHandler;`

Reply:
Tested the vanilla EH in local MP, it isn't broken by respawn and keeps working.
Zeus RC isn't an issue IMO, the EH also fires for RCd units when sighting in and updates `GVAR(visionMode)` correctly in any plausible scenario.

The CBA playerEH unfortunately doesn't fire when switching from WHOT to BHOT and can't differentiate between them. It also fires when sighting out, so launching unsighted despite previously selecting thermal would set the missile to normal view.